app/src/main/java/com/github/libretube/obj/FreetubeSubscription.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.libretube.obj
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class FreetubeSubscription(
+    @SerialName("name") val name: String,
```

Comment:
@SerialName("name") is not needed here as the variable name is already the same as the serial name.